packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -60,8 +60,17 @@
             {{/if}}
         
         "status":{{>ValueSet/ObservationStatus.hbs code=observationEntry.statusCode.code}},
-        "code": {{>DataType/CodeableConcept.hbs code=observationEntry.code text=observationEntry.text._}},
-        
+
+        {{#if observationEntry.code.code}}
+          "code": {{>DataType/CodeableConcept.hbs code=observationEntry.code text=observationEntry.text._}},
+        {{else}}
+          {{#if observationEntry.value.length}}
```

Comment:
i am not familiar with the length property of value. Is it actually in one of these:
https://build.fhir.org/ig/HL7/CDA-core-2.0/StructureDefinition-Observation-definitions.html#diff_Observation.value

Reply:
I applied the same logic as DiagnosticReport so removed the value logic